pokemongo_bot/cell_workers/nickname_pokemon.py

```diff
@@ -16,7 +16,8 @@ def work(self):
         else:
             pokemon_data = self._get_inventory_pokemon(inventory)
             for pokemon in pokemon_data:
-                self._nickname_pokemon(pokemon)
+                if not(pokemon.get('favorite', 0) is 1 and self.config.get('dont_nickname_favorite','')):
```

Comment:
since `dont_nickname_favorite` is a boolean, the `get` call should default to a boolean as well (`False`)